modules/install-sno-monitoring-the-installation-with-the-assisted-installer.adoc

```diff
@@ -0,0 +1,26 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="monitoring-the-installation-with-the-assisted-installer_{context}"]
+= Monitoring the installation with the Assisted Installer
+
+If you created the ISO using the Assisted Installer, use this procedure to monitor the installation.
+
+.Procedure
+
+. On the administration node, return to the browser and refresh the page. If necessary, reload the link:https://console.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer] page and select the cluster name.
```

Comment:
administration node - is it to formal? Maybe something like admin laptop? admin server? admin browser?

Reply:
here u called it admin node or laptop:
https://github.com/openshift/openshift-docs/pull/38068/files#diff-f90c1d082c6163fd5265e089a133d71aac2c1b833bf637ffcfc6ab4257d69d5fR10

Reply:
Or administration host maybe? Does "node" infer that it is part of the cluster?